app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
```

Comment:
- [ ] There is a bug that prevents a user from logging in.  I can sign up, but not login.  It asks me for the email confirmation, which I never receive, no matter how I try, or check the spam folder.

Please, kindly correct this issue.

Reply:
For this requirement, it can't send to the email for confirmation. The confirmation token can be found in the terminal when you create a user

Reply:
### The next reviewer
The confirmation token can be found in the terminal
Ctrl + click and the email will be confirmed

![confirmation-token](https://user-images.githubusercontent.com/61875592/164395976-4c03af41-5d33-4e4d-9390-861daaa64c40.PNG)

Reply:
Hey, @chukwuemeka1234 you are right.  I'll edit this one to OPTIONAL.